pkg/cli/main.go

```diff
@@ -173,8 +173,18 @@ func getSourceType(src string) (sourceType, error) {
 }
 
 func getReaderURL(src string) (vpkg.Reader, error) {
+	client := &http.Client{}
 
-	resp, err := http.Get(src)
+	req, err := http.NewRequest("GET", src, nil)
+	if err != nil {
+		return nil, err
+	}
+	fmt.Println(flagKey)
```

Comment:
Was this `fmt.Println` left in accidentally, or should this be a part of the commit?